src/libraries/System.Net.Security/src/System/Net/Security/SslCertificateTrust.cs

```diff
@@ -21,7 +21,7 @@ public static SslCertificateTrust CreateForX509Store(X509Store store, bool sendT
                 throw new PlatformNotSupportedException(SR.net_ssl_trust_store);
             }
 #else
-            if (sendTrustInHandshake)
+            if (sendTrustInHandshake && !System.OperatingSystem.IsLinux() && !System.OperatingSystem.IsMacOS())
             {
                 // to be removed when implemented.
                 throw new PlatformNotSupportedException("Not supported yet.");
```

Comment:
I think we should update the message and put it to resource file. This looks like omission from time when we expected to finish everything before 6.0 release